src/roaring.c

```diff
@@ -129,6 +129,46 @@ void roaring_bitmap_add_many(roaring_bitmap_t *r, size_t n_args,
     }
 }
 
+void roaring_bitmap_contains_multi(roaring_bitmap_t *r, size_t n_args,
+                             const uint32_t *vals, bool *results) {
+    if (n_args == 0 || r == NULL)  {
+        return;
+    }
+    uint32_t prev_container_key = 0;   // previous container key
```

Comment:
I think that prev_container_key is just the previous key... nothing to do with containers, so could be `prev_val`, for example. (Simpler)

Reply:
renew this key only when we went to another container.